slasher/detection/listeners.go

```diff
@@ -49,12 +49,14 @@ func (ds *Service) detectIncomingAttestations(ctx context.Context, ch chan *ethp
 	for {
 		select {
 		case indexedAtt := <-ch:
+			log.Debug("Running detection on incoming attestation")
 			slashings, err := ds.detectAttesterSlashings(ctx, indexedAtt)
 			if err != nil {
 				log.WithError(err).Error("Could not detect attester slashings")
 				continue
 			}
 			if len(slashings) < 1 {
+				log.Debug("Updating spans for incoming attestation")
```

Comment:
These logs are gonna be really spammy, maybe we should just log out the updating spans log?